### DISCUSSION TITLE
Flashing between consecutive stimuli with ITI = 0ms

### DISCUSSION THREAD

**Discussion by synapticlee** [jspsych/jsPsych#2950] [Q&A]
Hi, I have an experiment with no ITI, but when I test it, there is a quick flash between stimuli, instead of moving between them smoothly. I looked it up and there was a suggestion to use `on_canvas: false`, which I tried, but it didn't seem to fix the issue. Now, in the first block, I am having this problem, but in the other blocks, it seems to transition smoothly. Is this because of preloading images, perhaps?

**Comment by jodeleeuw** [comment]
Yes, this suggests that the images aren't preloading correctly. Are you using the preload plugin?

**Reply by synapticlee** [reply]
No, I was struggling to debug it. I tried just using `preload_images: images` in jsPsych.init (using 6.1.0) but that's when [my other issue](https://github.com/jspsych/jsPsych/discussions/2948#discussioncomment-4788898) (experiment failed to load) came up. Is there a more common way to preload images? I know there is a jsPsychPreload type..
